src/python/pants/backend/javascript/subsystems/nodejs.py

```diff
@@ -144,9 +218,15 @@ def npx(
         )
 
 
+@dataclass(frozen=True)
+class NodejsBinaries:
```

Comment:
Nit: The capitialization of `NodeJS` is inconsistent within this file: I see three variations.